runtime/config-deployer-service/ballerina/resources/apk-conf-schema.yaml

```diff
@@ -116,6 +116,7 @@ components:
           headerEnable:
             type: boolean
             default: true
+        additionalProperties: false
```

Comment:
just to know, what is the usage of this field?

Reply:
When we set `additionalProperties` to false, we are strictly allowing only the defined properties for that object. Otherwise the developer can define some random properties and there will be no validation errors.

@tharindu1st Shouldn't we add this to the VSCode schema as well?